features/admins/users/admin_fnok.feature

```diff
@@ -9,11 +9,11 @@ Feature: Admin Fannish Next Of Kind actions
       | login    | password   |
       | harrykim | diesalot   |
       | libby    | stillalive |
-      And I am logged in as a "policy_and_abuse" admin
+      And I am logged in as a "support" admin
```

Comment:
Should we be using scenario outlines instead so we can test every kind of admin (superadmin, support, policy_and_abuse) who is supposed to be able to edit FNOKs? Or am I just not seeing the tests that check that?

Reply:
This was missed, I'll add specs.